enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -313,8 +303,9 @@ private ClassPathResult buildClasspathFromException(
       if (cause instanceof ArtifactTransferException) {
         ArtifactTransferException artifactException = (ArtifactTransferException) cause;
         Artifact artifact = artifactException.getArtifact();
-        artifactProblems.add(
-            DependencyGraphBuilder.createUnresolvableArtifactProblem(dependencyGraph, artifact));
```

Comment:
I thought that the artifactProblems should be be passed to the second argument of ClassPathResult. But the field seems unused in this LinkageCheckerRule. Filed https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1392.